From 0eb1ee3db2f905c17757a06f77a5d79a351b3d33 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 18 Apr 2016 15:49:06 +0200
Subject: [PATCH] watchdog: clean return of installkernel()

return 0, otherwise if _alldrivers is empty, the return code is fail
---
 modules.d/04watchdog/module-setup.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/04watchdog/module-setup.sh b/modules.d/04watchdog/module-setup.sh
index 31102f4e..fbf6d6dd 100755
--- a/modules.d/04watchdog/module-setup.sh
+++ b/modules.d/04watchdog/module-setup.sh
@@ -73,4 +73,6 @@ installkernel() {
     # ensure that watchdog module is loaded as early as possible
     _alldrivers="${!_drivers[*]}"
     [[ $_alldrivers ]] && echo "rd.driver.pre=${_alldrivers// /,}" > ${initdir}/etc/cmdline.d/00-watchdog.conf
+
+    return 0
 }
